### ISSUE TITLE
inline functions inside JSX formatted incorrectly.

### ISSUE THREAD

**Issue by alinz** [maxogden/standard-format#158]
```js
const Foobar = (props) => {
  return (
    <button onClick={() => console.log('click')}>
    </button>
  )
}
```

transforms into

```js
const Foobar = (props) => {
  return (
  <button onClick={;() => console.log('click')}>
  </button>
  )
}
```

I undrestand that having `;` before function makes it safe when it minfies, but in this case, we don't need it.

**Comment by millermedeiros** [comment]
just released `esformatter-semicolon-first@1.2.0` with the fix!

**Comment by feross** [comment]
Nice, closing this now.
